### DISCUSSION TITLE
command line tools installation path

### DISCUSSION THREAD

**Discussion by stephenetyler** [Homebrew/discussions#5694] [Getting started] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.4.2
ORIGIN: https://github.com/Homebrew/brew
HEAD: 35ebf4a047b00b2b3de33ba0081d71293cdadede
Last commit: 3 days ago
Core tap JSON: 23 Oct 14:06 UTC
Core cask tap JSON: 23 Oct 14:06 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: vim
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.3.5 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.5/bin/ruby
CPU: octa-core 64-bit arm_blizzard_avalanche
Clang: 16.0.0 build 1600
Git: 2.39.5 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 15.0.1-arm64
CLT: 16.0.0.0.1.1724870825
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

  I have a fresh install of Mac 15 sequoia / m2 Mac.

I was prompted to upgrade to command line tools 16 via the OS.  I did that.

But when I looked.. I don't see it.
    ➜  ~ ls -al /Library/Developer/CommandLineTools/SDKs 

    total 0
    lrwxr-xr-x  1 root  wheel    14B Oct 23 10:20 MacOSX.sdk -> MacOSX15.0.sdk
    drwxr-xr-x  7 root  wheel   224B Oct 23 10:21 MacOSX14.5.sdk
    lrwxr-xr-x  1 root  wheel    14B Oct 23 10:19 MacOSX14.sdk -> MacOSX14.5.sdk
    drwxr-xr-x  7 root  wheel   224B Oct 23 10:20 MacOSX15.0.sdk
    lrwxr-xr-x  1 root  wheel    14B Oct 23 10:19 MacOSX15.sdk -> MacOSX15.0.sdk

  I'm really confused where command line tools 16 is installed.  I also manually installed the pkg file, with the same results.

Brew config shows that 16 is installed. - CLT: 16.0.0.0.1.1724870825

Is it safe to delete the old SDKs?  I guess they came preinstalled?

Thanks!

**Comment by Bo98** [comment, marked as answer]
CLT 16.0 comes with both MacOSX14.5.sdk and MacOSX15.0.sdk, so that looks correct to me.

**Reply by stephenetyler** [reply]
Ah I see.

I was expecting to see..
MacOSX16.sdk

That makes sense.

Sorry I'm relatively new to Mac.  I usually use Fedora.
